### PR TITLE
Documentation: Break out integration.md into different pages & add emojis to sidebar links

### PR DESCRIPTION
**Pull request by domyen**
Issue: na

## What I did
Integration.md was a very long page. I made it into 5 pages:
- Webpack
- Babel
- TypeScript
- Styling and CSS
- Images and assets

Also added emojis to the sidebar top level links.

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? this updates the docs

### CODE REVIEW

File: docs/toc.js [hunk 1]
```diff
@@ -155,9 +155,36 @@ module.exports = {
           type: 'link',
         },
         {
-          pathSegment: 'integration',
+          pathSegment: '',
```
Comment (kylesuss) [inline docs/toc.js:158]: Nice. Got this one 👌

**Review by kylesuss — APPROVED** [review]
LGTM. You usually intentionally leave no spacing in between the emoji & text, right? (Just clarifying as that is what is happening here).

**Comment by domyen** [comment]
I usually add a space between emoji and text.

**Comment by kylesuss** [comment]
@domyen you're all good. I see the space there now. I removed it in the markup and the spacing got even smaller. I think maybe there is some letter spacing at play which is making the emoji seem closer to the text than I was expecting.

**Pull request merged**